packages/core/js-client/src/services/_aqua/services.ts

```diff
@@ -49,113 +45,12 @@ export function registerSig(
   serviceId: string,
   service: Sig,
 ) {
+  const anyService: Record<never, unknown> = service;
```

Comment:
why is it named `anyService`?
```suggestion
  const sigService: Record<never, unknown> = service;
```

Reply:
btw, `registerSig` is named correctly, and the type `service: Sig` gives enough information so this function almost doesn't need a doc. Isn't it nice! :)

Reply:
Renamed all these `anyService`